community/cypher/frontend-3.0/src/main/scala/org/neo4j/cypher/internal/frontend/v3_0/helpers/fixedPoint.scala

```diff
@@ -17,20 +17,20 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-package org.neo4j.cypher.internal.compiler.v3_0.helpers.Converge
+package org.neo4j.cypher.internal.frontend.v3_0.helpers
```

Comment:
Any certain reason for why this is a `frontend` util rather than a `compiler` one?